test/mdn_test.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
```

Comment:
For the upcoming release we need to remove this import. It was an advisable optimization but only if you are running Python 3.7 or superior. Since the upcoming release will also accept Python 3.6, so that `sbi` can be used e.g. under Google Colab, this line must be deleted as it would produce an error when using Python 3.6.